pkg/bpf/map.go

```diff
@@ -106,6 +106,10 @@ type Map struct {
 	path string
 	once sync.Once
 	lock sync.RWMutex
+
+	// NonPersistent is true if the map does not contain persistent data
+	// and should be removed on startup.
```

Comment:
`pkg/bpf/map` is generic and not cilium specific at all. It is already being used by projects outside of Cilium. Using Cilium specific examples would be confusing. Is the description not clear?